System/Random.hs

```diff
@@ -574,12 +574,10 @@ runPrimGenIO_ g action = fst <$> runPrimGenIO g action
 type StdGen = SM.SMGen
 
 instance RandomGen StdGen where
-  next = SM.nextInt
```

Comment:
Not necessary - `SM.nextInt` just converted from `Word`, and that's what the default implementation does too.